### DISCUSSION TITLE
add . files error

### DISCUSSION THREAD

**Discussion by Vikassinghrathor** [community/community#42544] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

 can anyone help with this error warning: in the working copy of 'README.md', LF will be replaced by CRLF the next time Git touches it
error: 'client/' does not have a commit checked out
fatal: adding files failed

**Comment by jfullstackdev** [comment]
not very sure of that, are you referring to your local copy from GitHub ? can you show us the error ?

**Comment by getlost01** [comment]
The first error message you mentioned, "in the working copy of 'README.md', LF will be replaced by CRLF the next time Git touches it," suggests that Git is warning you that the line endings in your README.md file will be converted to a different format when it is next committed. This can occur if your repository is configured to use a different line ending style than the one used in your README.md file.

The second error message, "error: 'client/' does not have a commit checked out," suggests that you are trying to add files to a directory in your repository that does not have a commit checked out. In Git, a commit is a snapshot of the state of your repository at a particular point in time. In order to add files to a repository, you must have a commit checked out, which means that you must be on a branch and have a commit selected as the current commit.

The final error message, "fatal: adding files failed," suggests that the process of adding files to your repository has failed. This could be due to one of the issues mentioned above, or it could be due to some other problem with your repository.
